main.tf

```diff
@@ -43,7 +43,7 @@ resource "aws_ssm_activation" "default" {
 resource "aws_ssm_parameter" "activation_code" {
   for_each = toset(var.cluster_instances)
 
-  name        = "/ecs/${var.cluster_name}/activation/${each.value}/activation_code"
+  name        = "/ecs/activation/${each.value}/activation_code"
```

Comment:
This does mean that you have to know what instance names are already used for whatever cluster that already exists in the target account in order to prevent having doubles overwriting each other.

I think it would be better to keep the cluster name in the path to prevent any issues going forward (one could have many clusters defined in different locations of your code by different people).